tests/test_config.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=all
+import pathlib
+import pytest
+import os
+
+from uwtools.configure import Configure
+from uwtools.template import Template,TemplateConstants
+
+uwtools_file_base = os.path.join(os.path.dirname(__file__))
+
+def test_configuation_parse_env():
+
+    os.environ['TEST'] = 'TEST_TRUE'
+    config = Configure(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/experiment.yaml")))
+
+    expected = os.environ.get('TEST')
+    actual = config.test_env
+    assert actual == expected
```

Comment:
What happens when an environment variable is not set?

Reply:
Oh shoot I need to add another test for that, is is supposed to ignore the update and leave the ${KEY} in place